products/BellHybrid/apps/application-bell-main/ApplicationBellMain.cpp

```diff
@@ -33,6 +40,13 @@ namespace app
         : Application(name, parent, statusIndicators, startInBackground, stackDepth)
     {
         bus.channels.push_back(sys::BusChannel::ServiceDBNotifications);
+        bus.channels.push_back(sys::BusChannel::LayoutChangeRequests);
+
+        connect(typeid(gui::ChangeHomeScreenLayoutMessage), [&](sys::Message *msg) {
+            auto msgLayout = dynamic_cast<gui::ChangeHomeScreenLayoutMessage *>(msg);
+            setHomeScreenLayout(msgLayout->getLayoutName());
+            return sys::msgHandled();
+        });
```

Comment:
Why It's done by multicast channel since there is only one receiver? Shouldn't actions be used for such a purpose?

Reply:
Its my first case of app<->app communication and I got suggested to use channels mechanism. Shall I rewrite that part to use actions?

Reply:
reworked to action